fluentd/configs.d/user/fluent.conf

```diff
@@ -24,12 +24,6 @@
   @include configs.d/openshift/filter-syslog-record-transform.conf
   @include configs.d/openshift/filter-viaq-data-model.conf
   @include configs.d/openshift/filter-post-*.conf
-##
-
-## matches
-  @include configs.d/openshift/output-pre-*.conf
-  @include configs.d/openshift/output-operations.conf
-  @include configs.d/openshift/output-applications.conf
-  # no post - applications.conf matches everything left
-##
+  @include configs.d/openshift/pipeline-output-labels.conf
 </label>
+@include configs.d/openshift/pipeline-output-endpoints.conf
```

Comment:
We don't use this file anymore

Reply:
Which file?  fluent.conf?  This is a path forward where the image can be tested without dependency on CLO.  I believe I may need to add a few files back in for the case when PIPELINE is disabled

Reply:
Did you add a test to deploy fluentd using `fluentd/configs.d/user/fluent.conf` without using CLO?  If not, then `fluentd/configs.d/user/fluent.conf` is unused.